overlord/devicestate/handlers_install.go

```diff
@@ -1284,6 +1286,174 @@ func (m *DeviceManager) loadAndMountSystemLabelSnaps(systemLabel string) (
 	return sys, snapInfos, snapSeeds, mntPtForType, unmount, nil
 }
 
+func structureFromName(name string, Structures []gadget.VolumeStructure) (*gadget.VolumeStructure, error) {
```

Comment:
I'm used to search functions having the collection as first parameter, and the search query as the second one (cfr. `strstr()`).

Reply:
Another nitpick is `Structures` with the capital `S`, is there a reason for it?